Automattic/wp-calypso
# Use `connect_url`  to determine when to display Stripe Connect Screen

Relates to https://github.com/Automattic/gold/issues/72
Fixes https://github.com/Automattic/wp-calypso/issues/77898
To be tested with D114449-code

## Proposed Changes
Use `connect_url`  to determine when to display Stripe Connect Screen.


## Testing Instructions

Note: Ensure to sandbox `public-api.wordpress.com`

1. Go to `/earn/payments/${site_slug}`
2. Connect a Stripe account
3. Disconnect the Stripe account
4. Notice after disconnecting, the user prompted to reconnect



## Pre-merge Checklist
